app_feup/lib/controller/parsers/parser_print_balance.dart

```diff
@@ -13,3 +13,22 @@ Future<String> getPrintsBalance(http.Response response) async {
 
   return balance;
 }
+
+/// Extracts the print balance movements of the user's account
+///  from an HTTP [response].
+Future<List> getPrintMovements(http.Response response) async {
+  final document = parse(response.body);
+
+  final List<Map> movements = [];
+
+  final List rows =
+      document.querySelectorAll('table#tab_resultado > tbody > tr');
+
+  for (var row in rows) {
+    movements.add({
+      'datetime': row.children[0].innerHtml.trim().replaceAll('/', '-'),
+      'value': row.children[2].innerHtml.replaceAll('&nbsp;', ''),
+    });
+  }
+  return movements;
```

Comment:
Make use of .map() functions from List. It allows to create a List from another List like the example below
```suggestion
  final List rows =
      document.querySelectorAll('table#tab_resultado > tbody > tr');

  final List<Map> movements = rows.map((row) {
      return {
      'datetime': row.children[0].innerHtml.trim().replaceAll('/', '-'),
      'value': row.children[2].innerHtml.replaceAll('&nbsp;', ''),
      };
  }).toList();

  return movements;
```